js/utils/setTheme.js

```diff
@@ -35,14 +35,14 @@ module.exports = function(primaryColor, secondaryColor, backgroundColor, textCol
   }
 
   customStyleTag.innerHTML +=
-      "#ov1 #userPage .custCol-primary-light { transition: background-color .3s cubic-bezier(0, 0, 0.0, 1);  background-color: " + shadeColor2(primaryColor, 0.05) + ";}" +
```

Comment:
@morebrownies Mike, I removed the transitions from setThemes because it's pretty problematic for them to be there. If the transitions stayed and we end up applying those classes to anything that has an existing transition, the existing one will be overridden (basically, blown away). Since these color classes are so generic and can go on virtually any element, this can cause problems.

In my case, I needed a transition to be on modals, so that when chat is opened, the modal transitions into its newly centered position. With the generic nature of the color classes, I don't think there's any way for us to have both the transition on the colors and also maintain any existing transitions. It's arguably, a less than ideal aspect of css that there's not a way to say I want to add a transition on this property, but leave any existing transitions on other props in place.

Anyhow, to be honest, I don't really notice that much of a difference with the transition off. Let me know if you do and you think this is a big deal. I'm not sure exactly what we can do (aside from possibly a whole bunch of markup refactoring to ensure that the color classes never fall on elements with another transition in place), but we can definitely brainstorm and see if there's a way for us to eat our cake and have it too.